### ISSUE TITLE
Could not locate Comsol 5.3a installation

### ISSUE THREAD

**Issue by s-romeni** [MPh-py/MPh#105]
I have installed the module typing in the command prompt

```
pip install MPh
```

Then typed

```
python
>>> import mph
>>> client = mph.start(cores=1)
```

and I get the error `Could not locate any Comsol installation`. 

My Comsol installation (the executables `comsol.exe` and `comsolmphserver.exe`) is in `C:\Program Files\COMSOL\COMSOL53a\Multiphysics\bin\win64`, what should I do?

**Comment by john-hen** [comment]
Hi. You should upgrade to a more recent version of Comsol. Comsol 5.3a was released more than five years ago and is not supported, see [the documentation](https://mph.readthedocs.io/en/stable/installation.html):
> Versions 5.5 and newer are expected to work, i.e. have been tested.

That's because 5.5 was the current Comsol version back when MPh was first released. I believe MPh used to work with 5.3a for a while, so downgrading MPh might help, but I cannot be sure in which version the 5.3a support was "lost", if it was ever there, as that's not being tested. (See also issue #96, related to Comsol 5.3, released six month before 5.3a.)

**Comment by Nefteslon** [comment]
To launch 5.4, replacing the file "COMSOL\COMSOL54\Multiphysics\bin\win64\comsol.ini" with the same one, but from older versions, worked (maybe it will work for 5.3):
<details><summary>Details</summary>
<p>

# Configuration file for the Java bridge called from .NET
# '@user.dir' is evaluated to the same directory as this file
# '@user.home' is evaluated to the user's home directory
# '@os.home' is evaluated to the windows directory (e.g. 'c:\windows')
# '@comsol.root' is evaluated to the COMSOL root installation directory
-vm
..\..\java\win64\jre\bin\server\jvm.dll
-Djava.class.path=@comsol.root/plugins/com.comsol.webbridge_1.0.0.jar;@comsol.root/plugins/com.comsol.bridge_1.0.0.jar;@comsol.root/plugins/com.comsol.systemutils_1.0.0.jar;@comsol.root/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar;@comsol.root/plugins/org.eclipse.osgi_3.12.100.v20180210-1608.jar

-Dcs.root=@comsol.root
-Dcs.logoutput=file
-Dcs.logfileprefix=javabridge
-Dcs.singleprocess=on
-Dinstall.dir=@comsol.root
-Djava.security.manager=
-Djava.security.policy=@comsol.root/bin/win64/comsol.policy
-Dcs.javasecurity=on
-Xss4m
-Xms40m
-Xmx2g
-XX:MaxMetaspaceSize=512m

</p>
</details>
